tap_hubspot/__init__.py

```diff
@@ -89,9 +89,11 @@ class StateFields:
 
     "engagements_all":        "/engagements/v1/engagements/paged",
 
+    "contact_lists":        "/contacts/v1/lists",
```

Comment:
`contact_lists` and `contact_list` is confusing in what they mean. Could we have some other name for list of persons (contacts) in a given contact list? maybe `contacts`?

Reply:
I agree with you, it does look confusing. Sadly, there's an entity called `contacts` already. So I'm open to suggestions

Reply:
`contact_list_entries`?

Reply:
`contact_list_contacts` makes sense to me, thank you!
Addressed